cypress/integration/routes.spec.ts

```diff
@@ -0,0 +1,27 @@
+/// <reference types="cypress" />
+
+describe('Routes', () => {
+  it('visit all routes', () => {
+    cy.visit('/')
+
+    cy.findByText(/feedback/i).click()
+    cy.findByText(/Raids/i).click()
+    cy.get('h2')
```

Comment:
You could probably use Testing Library queries here too instead of `get` doesn't really matter either way though